lms/djangoapps/branding/tests/test_views.py

```diff
@@ -1,5 +1,6 @@
 # encoding: utf-8
 """Tests of Branding API views. """
+import copy
```

Comment:
since we are only using `deepcopy` change the import as 
`from copy import deepcopy`

Reply:
done